drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -280,12 +299,60 @@ def studio_submit(self, submissions, suffix=''):
         self.weight = float(submissions['weight'])
         self.item_background_color = submissions['item_background_color']
         self.item_text_color = submissions['item_text_color']
+        # Possible ValueError should be catched in _validate_submissions
+        self.max_items_per_zone = self._get_max_items(submissions)
         self.data = submissions['data']
 
         return {
             'result': 'success',
         }
 
+    @staticmethod
+    def _get_max_items(submission):
```

Comment:
@e-kolpakov For consistency, can we rename `submission` to `submissions` here? When I first read this I found myself double-checking if `studio_submit` was passing the `submissions` it receives unaltered, or if it was passing only a subset of the `submissions` data.